evap/staff/templates/staff_semester_view.html

```diff
@@ -421,9 +557,19 @@ <h3>
                                             </a>
                                         {% endif %}
                                         {% if course.can_be_deleted_by_manager %}
-                                            <button type="button" onclick="deleteCourseModalShow({{ course.id }}, '{{ course.name|escapejs }}');" class="btn btn-sm btn-outline-danger" data-bs-toggle="tooltip" data-bs-placement="top" title="{% trans 'Delete' %}">
-                                                <span class="fas fa-trash" aria-hidden="true"></span>
-                                            </button>
+                                            <confirmation-modal type="submit" form="course-deletion-form" name="course_id" value="{{ course.id }}" confirm-button-class="btn-danger">
+                                                <span slot="title">{% trans 'Delete course' %}</span>
+                                                <span slot="action-text">{% trans 'Delete course' %}</span>
+                                                <span slot="question">
+                                                    {% blocktrans trimmed with course_name=course.name %}
+                                                        Do you really want to delete the course <strong>{{ course_name }}</strong>?
+                                                    {% endblocktrans %}
+                                                </span>
+
+                                                <button slot="show-button" type="button" class="btn btn-sm btn-outline-danger" data-bs-toggle="tooltip" data-bs-placement="top" title="{% trans 'Delete' %}">
```

Comment:
(is already broken on main, we can make a new issue for this)
The button is rendered below the other buttons without any padding/margin for me:
![image](https://github.com/e-valuation/EvaP/assets/13838962/40b675fc-e280-4065-aeed-4e00568f28b6)

Reply:
It seems like it has to do with the sizing of the column itself and not with the modal style (giving the column `width: 20rem` makes all buttons display inline). So yeah, let's deal with this in another PR

Reply:
Moved to #2123.